src/v2/System/Router/PageLoader.tsx

```diff
@@ -73,7 +73,7 @@ export class PageLoader extends React.Component<
       <Box width="100%" style={style} className={className}>
         <ProgressBar
           percentComplete={progress}
-          highlight="purple100"
+          highlight="blue100"
```

Comment:
Should this be `brand`?

Reply:
I think this is outside the theme provider — brand wasn't working. Just being blue is fine in this case.